source/fundamentals/crud/read-operations/sort.txt

```diff
@@ -4,135 +4,382 @@ Sort Results
 
+The above ``sort()`` method returns a ``FindIterable`` object containing the
+documents in your collection, sorted from smallest to largest on the specified
+field name. 
+
+In the following code example, we use the ``ascending()`` method to sort the
+:ref:`sample collection <sorts-crud-sort-sample>`  
+by the ``_id`` field:
 
 .. code-block:: java
-  :emphasize-lines: 3
-    
-    // sort first by length, then title
-    MongoCursor<Document> cursor = collection.find()
-        .sort(ascending("length", "title")).iterator();
 
-This sample code prints your query results in ascending order by length. 
-When there are two documents with the same ``length`` value, it resolves the tie using the 
-``title`` field you specified as the second parameter.
+   import static com.mongodb.client.model.Sorts.ascending;
+
+   // <MongoCollection setup code here>
+   
+   List<Document> results = new ArrayList<>();
+   collection.find().sort(ascending("_id")).into(results);
+   for (Document result : results) {
+         System.out.println(result.toJson());
+   }
+
+The output of the code example above should look something like this: 
+
+.. code-block:: json
+
+   {"_id": 1, "letter": "c", "food": "coffee with milk"}
+   {"_id": 2, "letter": "a", "food": "donuts and coffee"}
+   {"_id": 3, "letter": "a", "food": "maple syrup"}
+   ...
+
+Descending
+~~~~~~~~~~
+
+To specify a descending sort, use the ``Sorts.descending()`` static factory
+method. Pass the ``Sorts.descending()`` method the name of the field you need to sort in descending order.
+
+The following code snippet shows how to specify a descending sort on the
+``_id`` field: 
 
 .. code-block:: java
+   
+   import static com.mongodb.client.model.Sorts.descending;
+
+   // <MongoCollection setup code here>
+   
+   collection.find().sort(descending("_id"));
+
+
+The code snippet above returns the documents in the
+:ref:`sample collection <sorts-crud-sort-sample>`  
+in descending order: 
+
+.. code-block:: json
 
-    Document{{_id=1, title=The Brothers Karamazov, length=824, author=Dostoyevsky}}
-    Document{{_id=5, title=Cryptonomicon, length=918, author=Stephenson}}
-    Document{{_id=3, title=Atlas Shrugged, length=1088, author=Rand}}
-    Document{{_id=6, title=A Dance with Dragons, length=1104, author=Martin}}
-    Document{{_id=4, title=Infinite Jest, length=1104, author=Wallace}}
-    Document{{_id=2, title=Les Misérables, length=1462, author=Hugo}}
+   {"_id": 6, "letter": "c", "food": "maple donut"}
+   {"_id": 5, "letter": "a", "food": "milk and cookies"}
+   {"_id": 4, "letter": "b", "food": "coffee with sugar"}
+   ...
 
-You can also sort by multiple fields in different orders, using ``Sorts.orderBy()``. 
-Consider this example where you first sort on ``length`` in an ascending order, and then on
-``title`` in a descending order:
+.. _sorts-crud-handling-ties:
+
+Handling Ties
+~~~~~~~~~~~~~
+
+A tie occurs when two or more documents have a field with identical values.
```

Comment:
I think it could be helpful for the reader if you further qualified the documents and field. First, the documents must be matching results on which the sort operates and second, the field must be one that determines the order. If that doesn't fit into a single sentence, maybe you can use a bulleted list to describe the details of each condition.

Reply:
Thanks for the note. After rereading, I can see how this sentence can be clearer by specifying the  more about the necessary conditions to produce a tie. I have added a clause specifying that a tie is tied to the field being used to order your results. Fix:

Going forward, I will try to refine my senses for spotting these unclear/incomplete statements.